docs/fundamentals/accessibility.md

```diff
@@ -0,0 +1,378 @@
+
+<!--
+### Semantic effects
+
+Link to semantic effects (.NET MAUI Community Toolkit) which combines `IsInAccessibleTree` and `ExcludedWithChildren` support, if docs are ever created, + SemanticOrderView.
+-->
+
+## Automation properties
+
+Automation properties are attached properties that can be added to any element to indicate how the element is reported to the underlying platform's accessibility framework.
+
+The `AutomationProperties` class defines the following attached properties:
+
+- `ExcludedWithChildren`, of type `bool?`, determines if an element and its children should be excluded from the accessibility tree. For more information, see [ExcludedWithChildren](#excludedwithchildren).
+- `IsInAccessibleTree`, of type `bool?`, indicates whether the element is available in the accessibility tree. For more information, see [IsInAccessibleTree](#isinaccessibletree).
+- `Name`, of type `string`, represents a short description of the element that serves as a speakable identifier for that element. For more information, see [Name](#name).
+- `HelpText`, of type `string`, represents a longer description of the element, which can be thought of as tooltip text that's associated with the element. For more information, see [HelpText](#helptext).
+- `LabeledBy`, of type `VisualElement`, which enables another element to define accessibility information for the current element. For more information, see [LabeledBy](#labeledby).
+
+These attached properties set native accessibility values so that a screen reader can speak about the element. <!-- For more information about attached properties, see [Attached Properties](~/xamarin-forms/xaml/attached-properties.md). -->
+
+> [!IMPORTANT]
+> Using the `AutomationProperties` attached properties may impact UI Test execution on Android. The `AutomationId`, `AutomationProperties.Name` and `AutomationProperties.HelpText` properties will both set the native `ContentDescription` property, with the `AutomationProperties.Name` and `AutomationProperties.HelpText` property values taking precedence over the `AutomationId` value (if both `AutomationProperties.Name` and `AutomationProperties.HelpText` are set, the values will be concatenated). This means that any tests looking for `AutomationId` will fail if `AutomationProperties.Name` or `AutomationProperties.HelpText` are also set on the element. In this scenario, UI Tests should be altered to look for the value of `AutomationProperties.Name` or `AutomationProperties.HelpText`, or a concatenation of both.
+
+Different screen readers read different accessibility values. For example, on Windows, Narrator will prioritize `AutomationProperties.Name`, `AutomationProperties.LabeledBy`, and then `AutomationProperties.HelpText`. On Android, TalkBack may combine the `AutomationProperties.Name` and `AutomationProperties.HelpText` values. Therefore, it's recommended that thorough accessibility testing is carried out on each platform to ensure an optimal experience.
```

Comment:
Would removing the example detract from the overall message? I'm not super confident that the priority order described is always true, but I'm also cool with leaving it if you have reason to believe it is!

Reply:
I think also noting here that the SemanticProperty APIs perform in a much more expected and consistent way between platforms might be useful as well.